### DISCUSSION TITLE
Test issue

### DISCUSSION THREAD

**Discussion by lilyminium** [openforcefield/discussions#4] [Show and tell]
This is just a test issue referencing other issues to see if conversion to a discussion and transferring back and forth retains references.

openforcefield/openff-toolkit#1487

https://github.com/openforcefield/openff-toolkit/issues/1487

**Comment by mattwthompson** [comment]
If we want to lean into this structure, we should consider turning off the discussions tabs within some repos (particularly the toolkit) and also disabling issues here. I think disabling PRs here would make it harder to tinker with the README and other files.

**Reply by lilyminium** [reply]
Thanks for the suggestion, Matt -- I've disabled discussions for a couple major repos, although the openff-bespoke-fit discussions seems to be somewhat in use. Unfortunately disabling issues means you can't transfer issues from e.g. the openff-toolkit to convert to discussions here, so I've left them on for now.
